packages/front-end/components/Share/ShareModal.tsx

```diff
@@ -0,0 +1,826 @@
+                <span
+                  className="drag-handle mr-2"
+                  {...provided.dragHandleProps}
+                >
+                  <GrDrag />
+                </span>
+                <h5 className="mb-0">{exp.name}</h5>
+                <div className="ml-auto">
+                  <span
+                    className="delete-exp cursor-pointer"
+                    onClick={(e) => {
+                      e.preventDefault();
+                      setSelectedExperiments(exp);
+                    }}
+                  >
+                    <FaRegTrashAlt />
+                  </span>
+                </div>
+              </div>
+            </div>
+          )}
+        </Draggable>
+      );
+      // adding options for each experiment... disabled for now
+      // expOptionsList.push(
+      //   <div>
+      //     {exp.name}
+      //     <div className="row">
+      //       <div className="col">
+      //         <div className="form-group form-check">
+      //           <label className="form-check-label">
+      //             <input
+      //               type="checkbox"
+      //               className="form-check-input"
+      //               checked={value.options[id]?.showScreenShots}
+      //               onChange={(e) => {
+      //                 const opt = { ...value.options };
+      //                 opt[id].showScreenShots = e.target.checked;
+      //                 const tmp = {
+      //                   ...value,
+      //                   options: opt,
+      //                 };
+      //                 manualUpdate(tmp);
+      //               }}
+      //               id="checkbox-showscreenshots"
+      //             />
+      //             Show screen shots (if avaliable)
+      //           </label>
+      //         </div>
+      //         <div className="form-row form-inline">
+      //           <div className="form-group">
+      //             <label className="mr-3">Graph type</label>
+      //             <select
+      //               className="form-control"
+      //               {...inputProps.options[id].graphType}
+      //             >
+      //               <option selected>{defaultGraph}</option>
+      //               <option>violin</option>
+      //             </select>
+      //           </div>
+      //         </div>
+      //       </div>
+      //     </div>
+      //   </div>
+      // );
+    });
+  }
+  const presThemes = [];
+  for (const [key, value] of Object.entries(presentationThemes)) {
+    if (value.show) {
+      presThemes.push(
+        <option value={key} key={key}>
+          {value.title}
+        </option>
+      );
+    }
+  }
+  //presThemes.push(<option value="custom">Custom</option>);
+
+  let previewUrl =
+    "preview?expIds=" +
+    value.experiments
+      .map((o) => {
+        return o.id;
+      })
+      .join(",") +
+    "&title=" +
+    value.title +
```

Comment:
Title may contain characters that aren't safe for URLs.  Make sure to urlencode them.  Same for description below.